### DISCUSSION TITLE
Github android app (Adding label work not correctly)

### DISCUSSION THREAD

**Discussion by litolax** [community/community#14846] [Mobile] [answered]
Issue: When I open issue on github repository with private adding labels permissions I can add all labels from andoid app. UI tell that label added but after update label disappears.

Expected behavior: UI must tell about lack of permissions.

**Comment by stevepopovich** [comment, marked as answer]
Hi @litolax, nice bug find! Filing and reporting to the team!

**Reply by Jay-o-Way** [reply]
I am helping for the PowerToys repo. I still have issues with permissions/tokens in the app. I should be able to set labels and more of such. App says i cannot. Is getting really annoying. Please fix ASAP!

**Reply by stevepopovich** [reply]
Hey Jay, you are having the same issue as above?

**Reply by stevepopovich** [reply]
The above issue should have been fixed in May

**Reply by Jay-o-Way** [reply]
> Hey Jay, you are having the same issue as above?

Yes, that's what i wanted to say

> The above issue should have been fixed in May

I'm saying it isn't 🙂 or it's a regression. Semantics?

**Reply by stevepopovich** [reply]
Hmm @Jay-o-Way, do you mind opening a new discussion with a video of what is happening? We can't repro this anymore.
